main.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap');
+
+:root{
+    --primary-color :#F65314;
+    --secondary-color :black;
+    --tertiary-color:#8da9bb;
+    --dark-color :#E50914; 
+    --light-color: white;
+    --yellow-shade-color :#FF9900;
+    --yellow-shade-color2: #ff7b25;
+    --green-shade-color1:#7CBB00;   
+    --green-shade-color2 :darkgreen;
```

Comment:
if the purpose of these variables is to provide light and a bit dark for color do the same as I mentioned above

Reply:
That's my personal preference, but thanks for your review.